tests/python/test_for_break.py

```diff
@@ -0,0 +1,23 @@
+import taichi as ti
+
+
+@ti.all_archs_with(print_ir=True)
+def test_for_break():
+    return
+    x = ti.var(ti.f32)
+
+    N = 16
+
+    ti.root.dense(ti.i, N).place(x)
+
+    @ti.kernel
+    def func():
+        for i in range(N):
+            for j in range(N + 5):
+                if j >= i:
+                    break
+                x[i] = j
```

Comment:
Maybe you want `x[i, j] = [i, j]` to track all assigns?

Reply:
Right, that makes the test stronger.